pkgs/applications/science/logic/coq/default.nix

```diff
@@ -37,6 +37,8 @@ self = stdenv.mkDerivation {
 
   passthru = {
     inherit coq-version;
+    inherit ocamlPackages;
+    # For compatibility
     inherit (ocamlPackages) ocaml camlp5 findlib num;
```

Comment:
Please do the corresponding cleaning in the various coq packages. Thanks.

Reply:
Good point. Done.